src/popup/router/routes.ts

```diff
@@ -442,6 +447,33 @@ export const routes: WalletAppRouteConfig[] = [
       showHeaderNavigation: true,
     },
   },
+  {
+    path: '/more/settings/secure-login',
+    component: SecureLoginSettings,
+    name: ROUTE_SECURE_LOGIN_SETTINGS,
+    meta: {
+      title: 'secureLogin',
+      showHeaderNavigation: true,
+    },
+    beforeEnter: (to, from, next) => {
```

Comment:
I think user should be able to open the secure login settings even if used device does not support it.
I strongly suggest to use composables in the route definitions.

Reply:
@onvisions What do you think?

Reply:
I think the user should not even see "Secure login" tab in Settings of the _browser extension_ because currently we do not support secure login for the extension, we do only for the apps. 
It will be a different story if we introduce the feature with password or PIN for the browser extension too. 

But currently it doesn't make sense to me: As a user I see the option "Secure login" in my browser extension Settings and I click because I'd like to have it enabled. I go to Secure login page and everything there is disabled. So now I am starting to wonder why... Is there a problem with my browser, OS. It's a waste of time and creates consfusion.

Another approach will be to display message that Secure login is currently supported only for the wallet mobile apps and not for the web version or browser extensions. However I think all this is an unnecessary complication of the feature and we'd better think of adding some secure authentication support for the web and extension users. @smaroudasunicorn

Reply:
I think I did wrong explanation so let me rephrase.

What I was suggesting is to not write any router level code that would block user from manually entering the Secure Login settings page as it makes sense because we are removing the button that allows tthe access to this page.

If user enters the page address of this settings page in the address bar he should access the page with an information that this feature is not available to him. Right now the `SecureLoginSettings` is not testing if user is actually able to use this feature which is bad in my opinion.

Also I forgot to add **no** to this sentence:
> I strongly suggest to use composables in the route definitions.

So it should be: I strongly suggest to **NOT** use composables in the route definitions.

Reply:
In this case I'll add the Secure login screen mockup for the case when the page is not accessible from Settings but still accessible directly via URL.I'm syncing it with Martin for the implementation.

Reply:
I wouldn't do anything complex. We have our `InfoBox` component so we need only some kind of text to be put there.